spec/latest/core/core.jsonld

```diff
@@ -505,6 +509,36 @@
       "domain": "hydra:IriTemplateMapping",
       "range": "xsd:string",
       "vs:term_status": "testing"
+    },
+    {
+      "@id": "hydra:offset",
+      "@type": "rdf:Property",
+      "label": "skip",
+      "comment": "Instructs to skip N elements of the set.",
+      "range": "xsd:nonNegativeInteger",
+      "vs:term_status": "testing"
+    },
+    {
+      "@id": "hydra:limit",
+      "@type": "rdf:Property",
+      "label": "take",
+      "comment": "Instructs to limit set only to N elements.",
+      "range": "xsd:nonNegativeInteger",
+      "vs:term_status": "testing"
+    },
+    {
+      "@id": "hydra:pageIndex",
+      "@type": "rdf:Property",
+      "label": "page index",
+      "comment": "Instructs to provide a specific page of the collection at a given index.",
+      "vs:term_status": "testing"
+    },
+    {
+      "@id": "hydra:pageReference",

```

Comment:
Is it only me or `pageReference` still does not make much sense? Can't we just go with `offset`, `limit` and `pageIndex` for now and create another issue or PR before we add it to the vocab.

Reply:
We wouldn't like to limit possible usages only to numeric pages - that's why `pageIndex`. I've recently added a `subPropertyOf` relation between `pageIndex` and `pageReference` - it feels natural this way.
Both predicates has similar semantics - here client you can point to a specific page of the set. The extra thing that comes with `pageIndex` is it's cleanness, which covers most use cases.
While it is not possible to provide i.e. min/max range or other means of expressing allowed values for `pageReference`, this predicate still can be used as it is now with some success.